src/scheduler/TaskScheduler.php

```
@@ -1,189 +1,195 @@
+        return $this->addTask($task, $delay, $period);
+    }
+
+    /**
+     * Cancel all scheduled tasks.
+     */
+    public function cancelAllTasks(): void {
+        foreach ($this->tasks as $task) {
+            $task->cancel();
+        }
+        $this->tasks->clear();
+        $this->queue->clear();
```

PHPStan (2.1.15)

Call to an undefined method pocketmine\utils\ReversePriorityQueue<int, pocketmine\scheduler\TaskHandler>::clear().


(method.notFound)